addons/web/static/src/libs/clipboard_polyfill.js

```diff
@@ -0,0 +1,100 @@
+/** @odoo-module **/
+
+class ClipboardItemImpl {
```

Comment:
I know @ged-odoo doesn't like the libs/ folder. I'm wondering where we would put this file if we didn't have that folder. It's not a lib. It exports nothing so it doesn't fit well in the core folder. And we do no want to see to many root files in the web/

Reply:
well in this case, it is not a lib, it is a polyfill. I would put it in `src/polyfills/clipboard.js` I guess. Not a big fan, but at least it's not lying